src/routes/search/containers/SearchPage.js

```diff
@@ -54,6 +83,50 @@ export class SearchPage extends React.Component<PropsType, LocalStateType> {
 
   onFilterTextChange = (filterText: string) => this.setState({filterText: filterText})
 
+  renderFeedback = (categories: Array<CategoryListItemType>) => {
```

Comment:
Probalby can be extracted to a separate component.

Reply:
Done